CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Experimental support for purls in PackageRecord and derived datastructures ([#414](https://github.com/mamba-org/rattler/pull/414))
 
+* Options to disable `bz2` and `zstd` in `fetch_repo_data`
```

Comment:
This should be in the Unreleased section. :) But you can also remove it, Ill add it when I make a new release.

Reply:
Woops, i moved it into the unreleased section